ui/urls.py

```diff
@@ -34,6 +34,8 @@
     url(r'^certificate/course/(?P<certificate_hash>[-\w.]+)?/?', CourseCertificateView.as_view(), name='certificate'),
     url(r'^certificate/program/(?P<certificate_hash>[-\w.]+)?/?', ProgramCertificateView.as_view(),
         name='program-certificate'),
+    url(r'^grade_records/(?P<record_hash>[-\w.]+)?/?', GradeRecordView.as_view(),
+        name='grade_records'),
```

Comment:
Can we use a URL pattern that's more similar to edX's? I think they do something like /records/programs/shared/:hashid/ 

I don't know what the significant of the "shared" part of the path is. And I guess they must include the program id in the hash somehow -- which I guess we're doing as well since the hash is unique to a particular enrollment.